mono/mini/mini-runtime.c

```diff
@@ -157,6 +157,24 @@ GSList *mono_interp_only_classes;
 
 static void register_icalls (void);
 
+static void
+print_stats_header (MonoStatsPrintPoint print_point) {
+	printf ("\n--------------------------------------------------------------------------------\n");
```

Comment:
Please use `g_print` instead of `printf`. While both have the same behavior on Desktop, it's different on e.g. Android: `printf` goes to `stdout` which is... somewhere, while `g_print` will be redirected the output to `adb logcat`.